sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -812,6 +812,17 @@ object SQLConf {
     .intConf
     .createWithDefault(65535)
 
+  val CODEGEN_METHOD_SPLIT_THRESHOLD = buildConf("spark.sql.codegen.methodSplitThreshold")
+    .internal()
+    .doc("The maximum source code length of a single Java function by codegen. When the " +
+      "generated Java function source code exceeds this threshold, it will be split into " +
+      "multiple small functions, each function length is spark.sql.codegen.methodSplitThreshold." +
```

Comment:
`each function length is spark.sql.codegen.methodSplitThreshold` this is not true, the method size is always larger than the threshold. cc @kiszk any idea about the naming and description of this config?

Reply:
IMHO, `spark.sql.codegen.methodSplitThreshold` can be used, but the description should be changed. For example,
`The threshold of source code length without comment of a single Java function by codegen to be split. When the generated Java function source code exceeds this threshold, it will be split into multiple small functions. ...`